src/artm/regularizer/smooth_time_in_topics_phi.h

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2017, Additive Regularization of Topic Models.
+
+   Author: Murat Apishev (great-mel@yandex.ru)
+
+   This class proceeds smoothing of tokens in Phi using nearest values.
+
+   More preferable for time stamps tokens. Requires to be used with sorted dictionary
+   (e.g. the tokens should follow in some order, chronological, for instance).
+
+   The formula of M-step is
+   
```

Comment:
Oh, you are right. I will resolve it in next way:
1) class_ids -> class_id -- I don't think, that using this regularizer with several modalities is correct
2) we suppose, that tokens of one modality are sorted, but we should hardcode restriction, that the are going as one block (e.g. that should be allowed to intersect with tokens of other modalities). To deal with it we need a minor modification of current regularization loop.